src/test/java/org/broadinstitute/hellbender/tools/exome/samplenamefinder/SampleNameFinderTest.java

```diff
@@ -0,0 +1,22 @@
+package org.broadinstitute.hellbender.tools.exome.samplenamefinder;
+
+import org.broadinstitute.hellbender.utils.test.BaseTest;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.util.List;
+
+public class SampleNameFinderTest extends BaseTest {
+    static final File INPUT_DIR = new File("src/test/resources/org/broadinstitute/hellbender/utils/segmenter/input/");
+    static final File INPUT_FILE = new File(INPUT_DIR, "HCC1143_reduced_log.tsv");
+
```

Comment:
maybe add some tests of exceptions for files with no names, tests that unique names are found when multiple names are present, etc.?